src/workerd/api/sockets.c++

```diff
@@ -209,10 +212,6 @@ jsg::Ref<Socket> connectImplNoOutputLock(
   JSG_REQUIRE(isValidHost(addressStr), TypeError,
       "Specified address is empty string, contains unsupported characters or is too long.");
 
-  auto& ioContext = IoContext::current();
```

Comment:
This looks like a policy change and, with only passing familiarity with this code, it would seem to be separate from the URL changes that make up the bulk here.

Reply:
Not so much a policy change as a minor perf improvement by moving the early exit fiddle check before we do any work here.

Reply:
For this, I think I'd rather keep this in this PR.